tests/tests/operation_tests.cpp

```diff
@@ -2336,6 +2336,347 @@ BOOST_AUTO_TEST_CASE( vesting_balance_withdraw_test )
+      generate_blocks(HARDFORK_615_TIME);
+      generate_block();
+      set_expiration( db, trx );
+
+      ACTORS((paul)(michael)(rachel)(alice));
+
+      // create assets
+      const auto& bitusd = create_bitasset("USDBIT", paul_id);
+      const auto& core   = asset_id_type()(db);
+      asset_id_type bitusd_id = bitusd.id;
+      asset_id_type core_id = core.id;
+
+      // fund accounts
+      transfer(committee_account, michael_id, asset( 100000000 ) );
+      transfer(committee_account, paul_id, asset(10000000));
+      transfer(committee_account, alice_id, asset(10000000));
+
+      // add a feed to asset
+      update_feed_producers( bitusd, {paul.id} );
+      price_feed current_feed;
+      current_feed.maintenance_collateral_ratio = 1750;
+      current_feed.maximum_short_squeeze_ratio = 1100;
+      current_feed.settlement_price = bitusd.amount( 100 ) / core.amount(5);
+      publish_feed( bitusd, paul, current_feed );
+
+      // paul gets some bitusd
+      auto call_paul = *borrow( paul, bitusd.amount(1000), asset(100));
+      call_order_id_type call_paul_id = call_paul.id;
+      BOOST_REQUIRE_EQUAL( get_balance( paul, bitusd ), 1000 );
+
+      // and transfer some to rachel
+      transfer(paul.id, rachel.id, asset(200, bitusd.id));
+
+      BOOST_CHECK_EQUAL(get_balance(rachel, core), 0);
+      BOOST_CHECK_EQUAL(get_balance(rachel, bitusd), 200);
+      BOOST_CHECK_EQUAL(get_balance(michael, bitusd), 0);
+      BOOST_CHECK_EQUAL(get_balance(michael, core), 100000000);
+
+      // michael selling core
+      auto call_michael = *borrow(michael, bitusd.amount(6), core.amount(8));
+      call_order_id_type call_michael_id = call_michael.id;
+
+      // add settle order and check rounding issue
+      force_settle(rachel, bitusd.amount(4));
+
+      BOOST_CHECK_EQUAL(get_balance(rachel, core), 0);
+      BOOST_CHECK_EQUAL(get_balance(rachel, bitusd), 196);
+      BOOST_CHECK_EQUAL(get_balance(michael, bitusd), 6);
+      BOOST_CHECK_EQUAL(get_balance(michael, core), 99999992);
+      BOOST_CHECK_EQUAL(get_balance(paul, core), 9999900);
+      BOOST_CHECK_EQUAL(get_balance(paul, bitusd), 800);
+
+      BOOST_CHECK_EQUAL( 1000, call_paul.debt.value );
+      BOOST_CHECK_EQUAL( 100, call_paul.collateral.value );
+      BOOST_CHECK_EQUAL( 6, call_michael.debt.value );
+      BOOST_CHECK_EQUAL( 8, call_michael.collateral.value );
+
+      generate_blocks( db.head_block_time() + fc::hours(20) );
+      set_expiration( db, trx );
+
+      // default feed and settlement expires at the same time
+      // adding new feed so we have valid price to exit
+      update_feed_producers( bitusd_id(db), {alice_id} );
+      current_feed.maintenance_collateral_ratio = 1750;
+      current_feed.maximum_short_squeeze_ratio = 1100;
+      current_feed.settlement_price = bitusd_id(db).amount( 100 ) / core_id(db).amount(5);
+      publish_feed( bitusd_id(db), account_id_type(19)(db), current_feed );
```

Comment:
Please replace `account_id_type(19)` with `paul_id`.

Reply:
LOL. Paul was the feed producer before this block. I didn't notice he lost his job.